src/Lesson8/HomeTask1.java

```diff
@@ -0,0 +1,37 @@
+package Lesson8;
+
+
+
+public class HomeTask1 {
+    int number;
+
+    public void method(int value) {
+        number = value;
+        if (number < 0) {
+            System.out.println("Number < 0");
+        }
+        if (number > 0) {
+            System.out.println("Number > 0");
+        }
+        if (number == 0) {
+            try {
+                throw new ArithmeticException("Error example");
+            } catch (ArithmeticException e) {
+                System.out.println("Exception");
+            }
```

Comment:
Here should NOT be any try/catch. You have just to throw an exception if number == 0;